source/MaterialXView/Viewer.cpp

```diff
@@ -1214,6 +1214,8 @@ void Viewer::loadDocument(const mx::FilePath& filename, mx::DocumentPtr librarie
 {
     // Set up read options.
     mx::XmlReadOptions readOptions;
+    readOptions.readComments = true;
```

Comment:
I'd recommend omitting these edits from the viewer, where we don't have the expectation that comments and blank lines in the original document will be maintained.